phpunit.xml.dist

```diff
@@ -18,15 +19,18 @@
 
     <php>
         <var name="driver_config_factory" value="Mink\WebdriverClassDriver\Tests\WebdriverClassicConfig::getInstance"/>
-        <!--server name="WEB_FIXTURES_HOST" value="http://test.mink.dev" /-->
+
+        <server name="WEB_FIXTURES_HOST" value="http://host.docker.internal:8002"/>
         <!-- MacOS -->
         <!--<server name="WEB_FIXTURES_HOST" value="http://docker.for.mac.localhost:8002"/>-->
-        <!--<server name="WEB_FIXTURES_BROWSER" value="firefox"/>-->
+
         <!-- where driver will connect to -->
         <server name="DRIVER_URL" value="http://localhost:4444/wd/hub"/>
+
         <!-- where DocumentRoot of 'Test Machine' is mounted to on 'Driver Machine' (only if these are 2 different machines) -->
-        <!--server name="DRIVER_MACHINE_BASE_PATH" value="" /-->
-        <!--server name="TEST_MACHINE_BASE_PATH" value="" /-->
+        <server name="DRIVER_MACHINE_BASE_PATH" value="/fixtures/"/>
+
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
```

Comment:
Changing the defaults of this section would result in the developer's inability to run the test suite with a locally started Selenium server.

Reply:
I replied to this point already, but I guess it got somehow lost:
https://github.com/minkphp/webdriver-classic-driver/pull/1#discussion_r1229075107

That still stands, the recommended test setup is with docker, therefore the defaults are tailored for docker. In case developers want to customize this, they can easily create their own `phpunit.xml` (that won't be committed with the project).